### PR TITLE
Bump doctrine/dbal dependency to 2.6.3

### PR DESCRIPTION
**Pull request by mikethebeer**
... so that `crate-dbal` can be used with PHP Symfony which requires dbal >= 2.6

### CODE REVIEW

File: src/Crate/DBAL/Platforms/CratePlatform.php [hunk 1]
```diff
@@ -102,9 +102,9 @@ public function supportsSequences()
     }
 
     /**
-     * If we want to support Schemas, we need to implement 
+     * If we want to support Schemas, we need to implement
      * getListNamespacesSQL and getCreateSchemaSQL methods
-     * 
+     *
```
Comment (seut) [inline src/Crate/DBAL/Platforms/CratePlatform.php]: as nothing besides formatting was changed inside this file, I'd prefer to revert the changes

File: composer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "prefer-stable": true,
     "require": {
         "php": "~5.5|~7.0",
-        "doctrine/dbal": "~2.5.5",
+        "doctrine/dbal": "~2.6.3",
```
Comment (seut) [inline composer.json]: any reason not to required just any 2.6 by `~2.6` ?

File: test/Crate/Test/DBAL/Platforms/CratePlatformTest.php [hunk 3]
```diff
@@ -67,25 +67,36 @@ public function getGenerateUniqueIndexSql()
...
-    
+
     public function getGenerateForeignKeySql()
     {
         $this->markTestSkipped('Platform does not support ADD FOREIGN KEY.');
     }
 
+    public function testGeneratesDecimalTypeDeclarationSQL()
+    {
+        $this->markTestSkipped('Platform does not support any decleration of datatype DECIMAL.');
```
Comment (seut) [inline test/Crate/Test/DBAL/Platforms/CratePlatformTest.php]: why is this related to the 2.6 upgrade?

File: test/Crate/Test/DBAL/Platforms/CratePlatformTest.php [hunk 4]
```diff
@@ -67,25 +67,36 @@ public function getGenerateUniqueIndexSql()
...
+    }
+
+    // Tests precision, scale, signed and unsigned on DOUBLE PRECISION
+    public function testGeneratesFloatDeclarationSQL()
+    {
+        $this->markTestSkipped('Platform does not support any decleration of datatype DOUBLE PRECISION.');
```
Comment (seut) [inline test/Crate/Test/DBAL/Platforms/CratePlatformTest.php]: also here, related to 2.6 upgrade?

File: composer.json (hunk 2 above)
Comment (mikethebeer) [reply composer.json]: @seut ~2.6.3 will allow all hotfix versions > 2.6.3 but < 2.7 so I'm not sure if I got your questions right.

File: test/Crate/Test/DBAL/Platforms/CratePlatformTest.php (hunk 3 above)
Comment (mikethebeer) [reply test/Crate/Test/DBAL/Platforms/CratePlatformTest.php]: this test method needs to get an override otherwise it would test datatypes `DECIMAL` which Crate does not support

File: test/Crate/Test/DBAL/Platforms/CratePlatformTest.php (hunk 4 above)
Comment (mikethebeer) [reply test/Crate/Test/DBAL/Platforms/CratePlatformTest.php]: same as above.

File: composer.json (hunk 2 above)
Comment (seut) [reply composer.json]: ah right, ignore ;)
